src/components/editor/tests/CodeEditor.spec.tsx

```diff
@@ -114,5 +114,9 @@ describe('CodeEditor', () => {
             expect(newList).not.toEqual(currentKeywords);
             expect(newList).toEqual(currentKeywords.concat(expectedNewKeywords));
         });
+
+        it('should have a border by default', () => {
+            expect(CodeEditor.defaultProps).toContain('mod-border');

```

Comment:
Your test isn't working. You're not checking the `className` prop. Maybe make it a little more complex like checking the prop when not sending `className` to make sure there is a default. You're only testing the content of a static variable for now.

Reply:
knowing how react works, isn't enough to test that?

Reply:
will add a test to look if the default class is added to the rendered code editor